test/vmapi.test.js

```diff
@@ -14,11 +14,22 @@ var libuuid = require('libuuid');
 function uuid() {
     return (libuuid.create());
 }
+var async = require('async');
```

Comment:
It appears that vasync is already included, and I also do not see any usages of async anywhere in this PR. Seems like that require can go away